installer/pkg/config-generator/ignition.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 const (
-	caPath = "generated/tls/root-ca.crt"
+	caPath  = "generated/tls/root-ca.crt"
+	tncPort = 49500
```

Comment:
rename to `ignServerPort`

Reply:
That's surprising. Is ignition what's effectively listening on this port?

Reply:
Also if we make this change, is there any point keeping the acronym TNC at all?

Reply:
I'd use `mcsPort`, since that is the actual service.

Reply:
Can we agree on doing this in another PR?

Reply:
I think it makes sense to change it here... since we are removing `*-tnc` from url.